### PR TITLE
Add StyledCollapse component to unify styles of Collapse component

### PR DESCRIPTION
**Pull request by loonywizard**
https://kupibiletru.atlassian.net/browse/MDT-329

### CODE REVIEW

**Comment by nezed** [comment]
New version of package: `4.3.1-feature-styled-collapse-ee5b6e72`

**Comment by nezed** [comment]
New version of package: `4.3.1-feature-styled-collapse-2bf9eed4`

**Comment by nezed** [comment]
New version of package: `4.3.1-feature-styled-collapse-cecbbefa`

**Comment by nezed** [comment]
New version of package: `4.3.1-feature-styled-collapse-f73abb9e`

File: src/components/StyledCollapse/index.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+  justify-content: space-between;
+  align-items: center;
+
+  padding: 12px ${({ withPadding }) => (withPadding ? '18px' : '0')};
+  width: 100%;
+  border-bottom: 1px solid ${({ theme }) => theme.color.miscLighter}
+`
+
+const PanelHeaderText = styled.div`
+  font-weight: bold;
```
Comment (privetkometa) [inline src/components/StyledCollapse/index.js]: 600

File: src/components/StyledCollapse/index.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
+/*
+ * StyledCollapse is just a simple wrapper of Collapse component
+ * It is used to unify collapse elements in the company
+ * */
+import React from 'react'
+import styled from 'styled-components'
+import Icon from 'components/Icon'
+import Collapse from 'components/Collapse'
```
Comment (privetkometa) [inline src/components/StyledCollapse/index.js]: import Collapse as StyledCollapse

File: src/components/StyledCollapse/index.js (hunk 1 above)
Comment (loonywizard) [reply src/components/StyledCollapse/index.js]: @privetkometa 700
https://developer.mozilla.org/ru/docs/Web/CSS/font-weight

File: src/components/StyledCollapse/index.js (hunk 2 above)
Comment (loonywizard) [reply src/components/StyledCollapse/index.js]: тут `as` не работает, можно только так
```import StyledCollapse from 'components/Collapse'```

**Comment by nezed** [comment]
New version of package: `4.3.1-feature-styled-collapse-37816877`

**Pull request merged**